Cargo.toml

```diff
@@ -48,3 +48,14 @@ resolver = "2"
 [patch."https://github.com/purestake/moonbeam"]
 pallet-author-mapping = { git = "https://github.com/zeitgeistpm/moonbeam", branch = "downgrade-staking-and-mapping" }
 parachain-staking = { git = "https://github.com/zeitgeistpm/moonbeam", branch = "downgrade-staking-and-mapping" }
+
+[profile.production]
+codegen-units = 1
+incremental = false
```

Comment:
Why is `incremental` set to false?

Reply:
> The incremental setting controls the [-C incremental flag](https://doc.rust-lang.org/rustc/codegen-options/index.html#incremental) which controls whether or not incremental compilation is enabled. Incremental compilation causes rustc to save additional information to disk which will be reused when recompiling the crate, improving re-compile times. The additional information is stored in the target directory.

It's default by the release profile [here](https://doc.rust-lang.org/cargo/reference/profiles.html#release), but Moonbeam wrote it still inside the configuration [here](https://github.com/PureStake/moonbeam/blob/eef06f9aba68412370130f2170bafdb45f4bd526/Cargo.toml#L44-L53)